config/v1/types_ingress.go

```diff
@@ -85,8 +85,75 @@ type IngressSpec struct {
 	// Note that if there are no RequiredHSTSPolicies, any HSTS Policy annotation on the route is valid.
 	// +optional
 	RequiredHSTSPolicies []RequiredHSTSPolicy `json:"requiredHSTSPolicies,omitempty"`
+
+	// LoadBalancer contains the AWS load balancer details
```

Comment:
Nit, as this is user facing, the comment should match the json tag not go type
```suggestion
	// loadBalancer contains the AWS load balancer details
```

While this struct currently only contains aws config, maybe we should expand the comment so we don't need to update it when we add new platforms in the future?